content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -253,6 +253,24 @@ fields that can be configured.
   confirmation window appears, as well.
 - :guilabel:`Analytic Account`: Select an analytic account to apply to this customer/quotation.
 
+When :doc:`Debug mode is enabled <../../../general/developer_mode>`, two additional fields appear in
+this section:
+
+- :guilabel:`Invoicing Journal`: The sales order invoices in the journal selected. When no selection
+  is made, the sales journal with the lowest sequence is used.
+- :guilabel:`Invoice Status`: The following options are available in this field:
```

Comment:
Hi @ksc-odoo - I only have two notes, since this is just adding that paragraph about being in the developer mode.  I do not see the second field you mentioned, so I cannot verify the information. I checked in runbots 17.0, 17.1, 17.2, and 17.4, and they all display this:
![image](https://github.com/user-attachments/assets/bc83fb6b-cb53-4d04-bb52-d2ccb7ab17ab)

I also tried to add products and other info to the quote to see if I could get the field to appear. So I am not sure if other fields must be populated in order to see that second new field or not?